packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -777,6 +778,99 @@ export async function launch({
       });
   }
 
+  const doCosmosInit = async (action, coreProposals) => {
```

Comment:
I think this should be named `doBootstrap`.

Cosmos init is currently overloaded to mean every start of the software.